tpm2-tss.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.27130.2020
+VisualStudioVersion = 15.0.27130.2027
```

Comment:
Gah. I've missed working with Visual Studo so much! Did a quick search to see if someone has come up with a clever way to keep VS from changing this field every time someone opens the file. I didn't find anything useful but I did find this: https://visualstudio.uservoice.com/forums/121579-visual-studio-ide/suggestions/13908807-do-not-automatically-change-the-visualstudioversio

Looks like people have reported this annoyance to Microsoft and the Visual Studio team. They just don't seem to care to fix it. For now it's not really an issue but if we start branching & merging a bunch I can see this file starting to thrash.